loadtest/control/simplecontroller/config.go

```diff
@@ -4,60 +4,50 @@
 package simplecontroller
 
 import (
-	"strings"
+	"encoding/json"
+	"fmt"
+	"os"
 
-	"github.com/mattermost/mattermost-load-test-ng/config"
-
-	"github.com/spf13/viper"
+	"github.com/mattermost/mattermost-load-test-ng/defaults"
 )
 
 // Config holds the the rate and user actions definitions that will be run by
 // the SimpleController.
 type Config struct {
 	// Actions are the user action definitions that will be run by the controller.
-	Actions []actionDefinition
+	Actions []actionDefinition `default_size:"1"`
```

Comment:
I think we should remove it from here. It's a tool helper but has nothing to do with the actual config default. I think it's okay for slices to be empty unless otherwise specified by the user.